pkg/services/authn/clients/jwt.go

```diff
@@ -114,6 +112,14 @@ func (s *JWT) Authenticate(ctx context.Context, r *authn.Request) (*authn.Identi
 	id.OrgRoles = orgRoles
 	id.IsGrafanaAdmin = isGrafanaAdmin
 
+	if s.cfg.JWTAuth.GroupsAttributePath != "" {
+		groups, err := s.extractGroups(claims)
```

Comment:
```suggestion
		groups, err := util.SearchJSONForStringSliceAttr(s.cfg.JWTAuth.GroupsAttributePath, claims)
```

Reply:
I dont think we need the extra function here. the only thing that one does is checking if GroupsAttributePath is empty and return and empty slice in that case. But we already do that check here

Reply:
if the attr is empty, it will return an err, prefer not to handle it